### ISSUE TITLE
Calendar.date(byAdding:value:to:) fails

### ISSUE THREAD

**Issue by BryanAamotAbbott** [skiptools/skip#182]
```
let startDate = Date(timeIntervalSince1970: 0)
let endDate = Calendar.current.date(byAdding: .day, value: 7, to: startDate)!

```
produces and end date of

1969-01-09 00:00:00 +0000

but it should be:

1970-01-08 00:00:00 +0000

**Comment by marcprux** [comment]
This is a bug in SkipFoundation: we were always wrapping the added components. Thanks for pointing it out. We should have a fix for it shortly.

**Comment by marcprux** [comment]
We've fixed this in skip-foundation 0.7.1. You should be able to run Xcode's `File`/`Packages`/`Update to Latest Package Versions` to get it. Please let us know if the issue persists after you update. Thanks!
